Add documention on how to handle CMORizers for multiple dataset versions (#2730)
ESMValGroup/ESMValTool: doc/sphinx/source/community/multiple_dataset_versions.rst

.. _dataset-versions:

> Support for several dataset versions

Reviewer: ```suggestion
Support for multiple versions of a dataset
```

************************************

> If you plan to update a CMORizer script to account for a newer version of a dataset,

Reviewer: ```suggestion
If you plan to update a CMORizer script to support a newer version of an existing dataset,
```

indicate in the issue or pull request if support for previous versions should be kept.
If the dataset is used in recipes, please also indicate if the recipes should be updated with
the newest dataset version.

> Policy to add and retire support

Reviewer: ```suggestion
Policy for dropping support for older dataset versions
```

================================

Support for older versions should preferably be kept as long as the data are publicly available.
This ensures reproducibility and eases comparison of results of recipes using this dataset.

> If previous dataset versions are no longer available or data issues have been fixed in a newer
> dataset version, it is preferable to only keep support for the latest version.

Reviewer: ```suggestion
Even when previous dataset versions are no longer available or data issues have been fixed in a newer
dataset version, it is preferable to keep support for the previous version in addition to supporting the newer version to allow for comparisons. In such cases, the recipe maintainers of recipes using the older version of the dataset should be asked to switch to the newer version if possible.
```
